src/cli/cli.cpp

```diff
@@ -1630,6 +1630,46 @@ void Interpreter::HandleDnsServiceResponse(otError aError, const otDnsServiceRes
 #endif // OPENTHREAD_CONFIG_DNS_CLIENT_ENABLE
 
 #if OPENTHREAD_FTD
+const char *EidCacheStateToString(otCacheEntryState aState)
+{
+    static const char *const kStateStrings[4] = {
+        "cache",
+        "snoop",
+        "query",
+        "retry",
+    };
+
+    return aState <= OT_CACHE_ENTRY_STATE_RETRY_QUERY ? kStateStrings[aState] : "unkno";
```

Comment:
```suggestion
    return aState <= OT_CACHE_ENTRY_STATE_RETRY_QUERY ? kStateStrings[aState] : "unknown";
```

Is it intentional ot use "unkno" ?

Reply:
It was intentional to keep the table aligned, but I don't think alignment is that important.